panther_analysis_tool/main.py

```diff
@@ -398,6 +399,88 @@ def upload_analysis(args: argparse.Namespace) -> Tuple[int, str]:
     return 0, ""
 
 
+def confirm_analysis_exists(args: argparse.Namespace, analysis_id_list: list) -> list:
+    validation_payload = {"listDetections": {"ids": analysis_id_list, "fields": ["id"]}}
+    client = get_client(args.aws_profile, "lambda")
+
+    validation = client.invoke(
+        FunctionName="panther-analysis-api",
+        InvocationType="RequestResponse",
+        LogType="None",
+        Payload=json.dumps(validation_payload),
+    )
+
+    api_response = json.loads(validation["Payload"].read().decode("utf-8"))
+    json_response = api_response["body"]
+
+    json_output = json.loads(json_response)
+
+    if len(analysis_id_list) != json_output["paging"]["totalItems"]:
```

Comment:
if lambda throws an error will this cause an exception here?

Reply:
Added some error handling